.github/ISSUE_TEMPLATE/controller-test-mac.md

```diff
@@ -10,10 +10,15 @@ assignees: ''
 #### Add Data
 Test to make sure Datavyu is able to load videos in all supported plugins. You may use the FrameCounter videos to test different resolutions/formats.
 
-- [ ] Open new instance of Datavyu.
-- [ ] On the Controller window, click Add Data.
-- [ ] Choose video file to open, On success a new video player window should open up.
-- [ ] Test adding multiple videos to the same Controller window, **Check to make sure the tracks area of the Controller window is properly populated with each stream**.
+- Open new instance of Datavyu.
+- On the Controller window, click Add Data.
+   - [ ] Success: File chooser window opens up above other windows. The starting folder is the folder containing the last video added to the Controller if there are any, otherwise the user's home folder. The window contains a plugin selection drop-down menu at the bottom of the screen that defaults to FFMPEG. Video files in the current directory are selectable.
```

Comment:
- We have issue templates for Mac OSX and for Windows, any update made to the controller-test-mac.md should be added to the controller-test-win.md. Please add update controller-test-win.md as well.
The file chooser must default to FFMPEG plugin but should provide Native OSX as an alternative on Mac. Please mention that here.

Reply:
Ok added to notes that Macs have additional plugin.